src/main/java/org/spongepowered/common/network/status/SpongeFavicon.java

```diff
@@ -155,4 +155,32 @@ private static BufferedImage decode(String encoded) throws IOException {
             base64.release();
         }
     }
+
+    public static final class FactoryImpl implements Favicon.Factory {
+
+        @Override
```

Comment:
Could you just move these static `load` methods into the Factory? seems like they're unused.

Reply:
Yeah, sure.